lms/templates/courseware/info.html

```diff
@@ -54,7 +54,7 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
 <section class="container">
   <div class="home">
     <div class="page-header-main">
-        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.id.org, course_name=course.id.course) | h}</h1>
+        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default) | h}</h1>
         <h2 class="page-subtitle">${course.display_name | h}</h2>
```

Comment:
Note that this line will still need the `| h` unless you also make the page safe by default by adding the page directive `<%page expression_filter="h" />`.  Note, if you do this, you'll need to check all cases of `${}` to ensure they require escaping.  If something really should support html, you could wrap it with the same `HTML()` method, but make sure you know that is what you want.

Also, I think there is a `course.display_name_with_default`.  Not sure if you want to use the same type of method here as well.